src/commonMain/kotlin/Account.kt

```diff
@@ -6,10 +6,28 @@ data class Account(
     val gender: String,
     val zipcode: Int,
     val username: String,
-    val email: String,
-    val password: String
```

Comment:
Note that this change prevents account passwords from being sent up to the frontend.  We can work with them in the backend (something I would like to make more difficult to do in the future).

Reply:
good call